src/osvr/Client/EyeTrackerRemoteFactory.h

```diff
@@ -0,0 +1,69 @@
+/** @file
+    @brief Header
+
+    @date 2015
+
+    @author
+    Sensics, Inc.
+    <http://sensics.com/osvr>
+
+*/
+
+// Copyright 2015 Sensics, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef INCLUDED_EyeTrackerRemoteFactory_h_GUID_6788A8FD_F38F_419F_D2F2_6FCA976CADB4
+#define INCLUDED_EyeTrackerRemoteFactory_h_GUID_6788A8FD_F38F_419F_D2F2_6FCA976CADB4
+
+// Internal Includes
+#include "VRPNConnectionCollection.h"
+#include <osvr/Common/InterfaceList.h>
+#include <osvr/Common/OriginalSource.h>
+#include <osvr/Util/SharedPtr.h>
+#include "RemoteHandler.h"
+#include <osvr/Common/ClientContext.h>
+
+// Library/third-party includes
+// - none
+
+// Standard includes
+// - none
+
+namespace osvr {
+namespace client {
+
+    class EyeTrackerRemoteFactory {
+      public:
+        EyeTrackerRemoteFactory(VRPNConnectionCollection const &conns);
+
+        template <typename T> void registerWith(T &factory) const {
+            factory.addFactory("eyetracker", *this);
+            factory.addFactory("location2D", *this);
```

Comment:
I see, so since the location2D and direction can be reported independently from eyetracker, they aren't needed. But for the tracker the pose, position and orientation are dependent on it, right?